Makefile

```diff
@@ -16,97 +16,103 @@ METADATA_DEFAULTS_FILE=${MAKEFILE_PATH}/pkg/config/defaults/aemm-metadata-defaul
 ENCODED_METADATA_DEFAULTS=$(shell cat ${METADATA_DEFAULTS_FILE} | base64 | tr -d \\n)
 DEFAULT_VALUES_VAR=github.com/aws/amazon-ec2-metadata-mock/pkg/config/defaults.encodedDefaultValues
 
+version:
+	@echo ${VERSION}
+
+latest-tag:
+	@echo ${LATEST_TAG}
+
+image:
+	@echo ${IMG_W_TAG}
+
 create-build-dir:
 	mkdir -p ${BUILD_DIR_PATH}
 
 clean:
 	rm -rf ${BUILD_DIR_PATH}
 
-fmt:
-	goimports -w ./
-
 compile:
 	@echo ${MAKEFILE_PATH}
 	go build -a -tags aemm${GOOS} -ldflags '-X "${DEFAULT_VALUES_VAR}=${ENCODED_METADATA_DEFAULTS}"' -o ${BUILD_DIR_PATH}/${BINARY_NAME} ${MAKEFILE_PATH}/cmd/amazon-ec2-metadata-mock.go
 
-build: create-build-dir validate-json compile
+validate-json:
+	${MAKEFILE_PATH}/test/json-validator
 
-build-and-test: build test
+build: create-build-dir validate-json compile
 
-build-binaries:
-	${MAKEFILE_PATH}/scripts/build-binaries -d -p ${SUPPORTED_PLATFORMS} -v ${VERSION}
+unit-test: create-build-dir
+	go test ${MAKEFILE_PATH}/... -v -coverprofile=coverage.txt -covermode=atomic -outputdir=${BUILD_DIR_PATH}
```

Comment:
can you modify this target to support benchmarks and change the coverage file extension to out: 
	
```
go test -bench=. ${MAKEFILE_PATH}/...  -v -coverprofile=coverage.out -covermode=atomic -outputdir=${BUILD_DIR_PATH}
```